### PR TITLE
Updated Footer Color from #101820 to #231B4E

### LINKED ISSUE #130: Code: Update the Footer Color
### Is your feature request related to a problem? Please describe.
The footer color has to be changed so that it follows the AnitaB Color Guidelines.

### Describe the solution you'd like
The existing color code should be changed to #231b4e. 
The footer color inspiration has been taken from [GHC](https://ghc.anitab.org/speakers/)

### PR DESCRIPTION
**Pull request by adithyaakrishna**
Issue #130
Update The Footer Color

### Description

The color code of the footer was changed from #101820 to #231B4E to follow the AnitaB Color Guidelines.

Fixes #130  [ISSUE]

### Type of Change:

- Code
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by adithyaakrishna** [comment]
The most recent commit from me, commit no. 6de9964 added a config.yml file for the welcome bot as requested in Issue #131 
The file is placed in .github directory to allow for easier installation and setup of the Welcome-Bot

**Comment by adithyaakrishna** [comment]
Thank You, I saw that there are branches, but didn't know to which branch I can commit.
Let me know if Can I create another branch :)
I've made the changes too

**Comment by vj-codes** [comment]
@adithyaakrishna you can create a  new branch on your fork and then push the changes to develop branch .
For example for the config.yml file , you could create a new branch named as config and push by `git push origin config`.
That's what I meant, but since this PR is already created dont change any branch for now. Hope you got what I meant!
Also I dont see the new changes any idea why?

**Comment by adithyaakrishna** [comment]
My bad, I forgot to push xD

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,13863 @@
+{
```
Comment (annabauza) [inline package-lock.json:1]: this file is not needed we use yarn not npm

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Configuration for the-welcome-bot - https://github.com/bash-bot/the-welcome-bot
```
Comment (annabauza) [inline .github/config.yml:1]: this change  should not be committed yaml is configuration not documentation file...

**Pull request merged**